src/test/bench_test/config.h

```diff
@@ -50,6 +48,12 @@ struct config : public ::dsn::utils::singleton<config>
     uint64_t seed;
     // Default environment suitable for the current operating system
     rocksdb::Env *env;
+
+private:
+    config();
+    ~config() = default;
```

Comment:
should we expose a `delete` interface?

Reply:
Sorry I have no opinion about what you mean